components/markets/SellFullSetForm.tsx

```diff
@@ -47,9 +45,12 @@ const SellFullSetForm = ({
   const [amount, setAmount] = useState<string>("0");
   const [maxTokenSet, setMaxTokenSet] = useState<Decimal>(new Decimal(0));
 
-  const colors = market?.categories
-    ? calcMarketColors(marketId, market.categories.length)
-    : [];
+  const extrinsicBase = wallet.activeAccount?.address
+    ? sdk
+        ?.asRpc()
+        .api.tx.balances.transfer(wallet.activeAccount?.address, ZTG.toFixed(0))
```

Comment:
@Robiquet Same as the other comments, should we transfer on behalf of `wallet.readAddress` when using proxies?